app/jenkins-for-jira-ui/.eslintrc.json

```diff
@@ -85,7 +85,8 @@
 					":": "before"
 				}
 			}
-		]
+		],
+		"import/no-extraneous-dependencies": "warn"
```

Comment:
<img width="1315" alt="Screenshot 2023-11-16 at 2 04 17 pm" src="https://github.com/atlassian/jenkins-for-jira/assets/37155488/02603ce9-a02f-4e7f-8c83-341c97e498e6">

Started seeing this nonsense when committing.

Reply:
I find that warnings get ignored. Would either of these solutions work better for us? They will omit this specific dev dependencies error while allowing us to catch any other violations of this rule :) 

```
"import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
```

or 

Disabling it specifically for test files
```
"import/no-extraneous-dependencies":[
  "error",
  {
     "devDependencies":[
        "**/*.test.ts",
        "**/*.test.tsx"
     ]
  }
]
```

Source: https://stackoverflow.com/questions/44939304/eslint-should-be-listed-in-the-projects-dependencies-not-devdependencies